src/implementations/config/create.ts

```diff
@@ -1,19 +1,19 @@
+import { utils, v2, v3 } from "@govtechsg/open-attestation";
 import fs from "fs";
 import path from "path";
 import { info } from "signale";
-import { readFile } from "../../implementations/utils/disk";
-import { handler as createTemporaryDns } from "../../commands/dns/txt-record/create";
 import { CreateConfigCommand } from "../../commands/config/config.type";
-import { Dns } from "./types";
+import { handler as createTemporaryDns } from "../../commands/dns/txt-record/create";
+import { readFile } from "../../implementations/utils/disk";
```

Comment:
This import could be just this:

```suggestion
import { readFile } from "../utils/disk";
```

Reply:
ok